src/cpp/r/session/REmbeddedWin32.cpp

```diff
@@ -301,8 +227,14 @@ void runEmbeddedR(const core::FilePath& rHome,
    // no signal handlers (see comment in REmbeddedPosix.cpp for rationale)
    R_SignalHandlers = 0;
 
-   // initialize R_max_memory
-   initializeMaxMemory(rHome);
+   // setup command line options
+   // note that R does a lot of initialization here that's not accessible
+   // in any other way; e.g. the default translation domain is set within
+   //
+   // https://github.com/rstudio/rstudio/issues/10308
+   static const int rargc = 2;
+   static const char* rargv[] = { "R.exe", "--vanilla" };
+   ::cmdlineoptions(rargc, (char**) rargv);
```

Comment:
Are these arguments consumed by R in some way or do we just need to pass in placeholders? (seems like if they are actually consumed it could lead to e.g. R not running startup scripts, and/or getting confused on non-Windows platforms?)

Reply:
They are read + consumed, and are used to set some default values for the R startup parameters; e.g. things like here:

https://github.com/wch/r-source/blob/e3d6c3fdd8666ae96bee913e1c0edab7bbaa088d/src/gnuwin32/system.c#L1165-L1176

But we do later set the parameters ourselves here:

https://github.com/rstudio/rstudio/blob/9d3b5a4f4c5abbceee1e91efb7bb90ec4727018d/src/cpp/r/session/REmbeddedWin32.cpp#L336-L370

I think I need to remove `--vanilla` here since we do want to process Renviron / Rprofile and such, though. I think I'm going to do one more pass over `cmdlineoptions()` to see if it does anything that we need to undo afterwards.